### ISSUE TITLE
Change book title in PDF (digital) from H1 to P or Title

### ISSUE THREAD

**Issue by MichSpieler** [pressbooks/pressbooks#4298]
In PDF (digital) exports, the book title is tagged as an H1. Since Part and Chapter titles are also tagged as H1, this conflicts with accessibility guidelines.

The [PDFA guidelines](https://pdfa.org/resource/tagged-pdf-best-practice-guide-syntax/) state three examples when multiple H1s are acceptable:

> Example A:
> ```
> <H1> [first heading, no title present]
> <P>
> <H2>
> <P>
> ```
> Example B:
> ```
> <P> [encloses document title]
> <H1> [first main heading, e.g.”1. Top level heading”]
> <P>
> <H2> [heading, 2nd level, e.g. “1.1 2nd level heading”]
> <P>
> ```
> Example C:
> ```
> <Title> [encloses document title (role-mapped to <P>)]
> <H1> [first main heading]
> <P>
> <H2>
> <P>
> ```
> Example D:
> ```
> <H1> [encloses document title]
> <H2> [first main heading]
> <P>
> <H2>
> <P>
> In examples A, B and C, it is semantically acceptable to use the <H1> headings several times. Example D, a case of <H1> used for a title (which is not a violation of PDF/UA-1), is only semantically appropriate if there is only one <H1> in the document.  

Examples B or C seem most relevant for our case. The title does appear twice, once on the [title page](https://github.com/pressbooks/pressbooks/blob/dev/templates/export/title.blade.php) and once on the [half-title page](https://github.com/pressbooks/pressbooks/blob/dev/templates/export/half-title.blade.php). 
Both should not be tagged as titles, so one option would be to change title.blade.php to
```
<Title class="title">{!! $title !!}</Title>
<p class="subtitle">{!! $subtitle !!}</p>
```
and half-title.php to
```
<div id="half-title-page"><p class="title">{!! $title !!}</p></div>
```

**Comment by MichSpieler** [comment]
From bug scrub: let's ask @greatislander which example (B or C) would be better to use.

**Comment by MichSpieler** [comment]
From meeting with Ned: we should look at this together with other heading related issues and look at how headings are handled in different formats. Mich will create an Epic and link to the existing issues.
